src/ril_plugin.c

```diff
@@ -27,6 +27,7 @@
 #include "ril_vendor.h"
 #include "ril_devmon.h"
 #include "ril_log.h"
+#include "ril-constants.h"
```

Comment:
```
src/ril_plugin.c:30:10: fatal error: ril-constants.h: No such file or directory
 #include "ril-constants.h"
          ^~~~~~~~~~~~~~~~~
compilation terminated.
```
I think you can just drop this line, this header must be included from elsewhere as `<ofono/ril-constants.h>`